packages/mongodb-memory-server-core/src/util/db_util.ts

```diff
@@ -31,4 +31,16 @@ export function isNullOrUndefined(val: unknown): val is null | undefined {
   return val === null || val === undefined;
 }
 
+/**
+ * Assert an condition, if "false" throw error
+ * Note: it is not named "assert" to differentiate between node and jest types
+ * @param cond The Condition to throw
+ * @param error An Custom Error to throw
+ */
+export function assertion(cond: unknown, error?: Error): asserts cond {
```

Comment:
In most repos (especially from Facebook) I saw `invariant` name for the method of such kind
https://github.com/graphql/graphql-js/blob/master/src/jsutils/invariant.js

I don't know but maybe you saw `assertion` name in some other OSS repos which are used not in test suites.

Reply:
actually, i didnt read that somewhere - i created that function while reviewing the new `asserts` keyword in typescript (i think 3.7 or 3.8 added this), but because nodejs's assert didnt include this (and still dosnt), i had to create an new function and to not confuse (or accidentally import the wrong one) i named it `assertion`, instead of `assert` (and i didnt use nodejs's assert in that function, because i also tried to make typegoose "brower-ready" (running without nodejs))

PS: your linked file / function dosnt have the keyword `assertion`

Reply:
Yep, instead `assertion` name in the majority of repos is used `invariant` name. 
For example https://www.npmjs.com/package/invariant
Mostly it's used for code modification in PROD env in browsers. But we are on the server side and don't have aims to minify our code. So let keep this method with your name `assertion`.